src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+https://github.com/catalystneuro/neuroconv@add_ignore_timestamps_errors_to_interface#egg=neuroconv
+git+https://github.com/catalystneuro/neuroconv.git@36fe8ba02c036e715a915ee66079391a50536fb2#egg=neuroconv
```

Comment:
@CodyCBakerPhD I added here, should I also add to `fiber_photometry` requirements as well?

Reply:
@weiglszonja Probably; in any situation where there is a combination of `main` of `neuroconv` and `ndx-fiber-photometry`

Reply:
Added there too until https://github.com/catalystneuro/ndx-photometry/issues/8 is resolved (Ryan is on it!)

Reply:
good to go @CodyCBakerPhD